types/ioTracker/ioTracker3/default.schema.json

```diff
@@ -0,0 +1,61 @@
+{
+  "$id": "https://akenza.io/ioTracker/ioTracker3/default.schema.json",
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "title": "Default",
+  "type": "object",
+  "properties": {
+    "uplinkReason": {
+      "title": "Uplink reason",
+      "description": "Describes why a message got sent",
+      "type": "string",
+      "enum": ["BUTTON", "MOVEMENT", "GPIO"],
+      "hideFromKpis": true
+    },
+    "crc": {
+      "title": "Cyclic redundancy check",
+      "description": "Cyclic redundancy check",
+      "type": "integer",
+      "hideFromKpis": true
+    },
+    "batteryLevel": {
+      "type": "integer",
+      "minimum": 0,
+      "maximum": 100,
+      "title": "Battery charge",
+      "description": "The battery charge in percent.",
+      "unit": "%"
+    },
+    "temperature": {
+      "title": "Temperature",
+      "unit": "°C",
+      "type": "number",
+      "description": "The temperature in °C.",
+      "minimum": -3276.5,
+      "maximum": 3276.5
+    },
+    "light": {
+      "title": "Light",
+      "unit": "lux",
+      "type": "number",
+      "description": "The light in lux.",
+      "minimum": 0,
+      "maximum": 65535
+    },
+    "maxAccelerationNew": {
```

Comment:
why new?

Reply:
Well its the new max acceleration in comparison to the old

Reply:
so it is the difference?